apps/freelance/src/pages/ProfileQuestions/model.ts

```diff
@@ -1,13 +1,17 @@
 import { Moment } from 'moment';
 
-export interface IProfileQuestions1 {
+export interface IProfileQuestions {
   [key: string]: string | number | object | undefined;
```

Comment:
`[key: string]?: string | number | object;` should work

Reply:
fixed